### PR TITLE
Version bump of admin and chosen-rails gems

### PR DESCRIPTION
**Pull request by bishboria**
Due to a vulnerability with jquery-rails. See See
https://groups.google.com/forum/#!topic/rubyonrails-security/XIZPbobuwaY
for details
- Change to config/application.rb is a reported issue rails/sass-rails#315
- chosen-rails interacts with sass so have to upgrade it too

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -48,11 +48,11 @@ else
   gem 'gds-sso', '10.0.0'
 end
 
-gem 'govuk_admin_template', '1.4.0'
+gem 'govuk_admin_template', '2.3.1'
```
Comment (boffbowsh) [inline Gemfile]: If bumping 1.x to 2.x, please remove GA code, see https://github.com/alphagov/travel-advice-publisher/pull/81 as an example

**Comment by boffbowsh** [comment]
Please add https://gist.github.com/boffbowsh/073d4f36f8677304fa9f too

**Comment by alext** [comment]
@boffbowsh The hotfix has already been taken of in #256

**Comment by rboulton** [comment]
Stuart's now removed the GA code => merging.

**Pull request merged**